public/index.html

```diff
@@ -5,6 +5,10 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Groupon-App: Customer Review Component</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600|Roboto&display=swap" rel="stylesheet">
```

Comment:
@d-rowe , do you know of an easy way to import this directly into React? Not sure if I should be using links directly in the html file.

Same goes for the Material Icons font a few lines down.

Reply:
If you'd like it looks like you could place this in your styled component in your top level component
`@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');`
`font-family: 'Roboto', sans-serif;`